docs/development.md

```diff
@@ -8,7 +8,7 @@ development. To use it first do the following:
 $ make dev-image
 $ # The build will take a while...
 $ mkdir local-etc && cp packaging/etc/agent.yaml local-etc/  # Get a basic operating config file in place
-$ make run-dev-image
+$ AGENT_BIN=/bundle/bin/signalfx-agent make run-dev-image
```

Comment:
`AGENT_BIN` is not required when starting the dev image.  It is only required when running the integration tests in the dev image to override the default (https://github.com/signalfx/signalfx-agent/blob/main/Dockerfile.dev#L30).

Reply:
It may be better to update https://github.com/signalfx/signalfx-agent/blob/main/tests/README.md?plain=1#L34-L35 instead.